client/src/Component/Common/Banner/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import logo from '../../../assets/logo2.svg';
+import {
+  BackgroundWrapper,
+  TitleText,
+  TitleTextSub,
+  BannerLogo,
+  SubtitleText,
+} from './index.style';
+
+const BannerTitle = ({ title, subtitle }) => {
```

Comment:
you need to add props validation to `title`, `subtitle `, using `'prop-types'`, see Header component to refer

Reply:
No worries, we just did. We have actually never used or heard of prop-types before so we had a look into it and have implemented it in the component now! :)